packages/materialized-views/src/generate-content.js

```diff
@@ -10,16 +10,16 @@ const getRealms = async (repos) => {
 
 const { log } = console;
 
-export default async () => {
+export default async ({ granularity }) => {
   const tenants = await getAllTenants();
 
   const now = dayjs();
   const limit = 250;
-  const start = now.subtract(1, 'week').startOf('day').toDate();
+  const start = now.subtract(1, granularity).startOf('day').toDate();
   const end = now.endOf('day').toDate();
 
   await eachSeries(tenants, async (slug) => {
-    log(`Updating weekly content data for ${slug}...`);
+    log(`Updating ${granularity}ly content data for ${slug}...`);
```

Comment:
Perhaps do `Updating weekly content data for ${slug} by ${granularity}...` instead?